metaverse_tools/.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "python.pythonPath": "C:\\Users\\Malactus\\AppData\\Local\\Programs\\Python\\Python39\\python.exe"
```

Comment:
This file should be added to the .gitignore probably.

Reply:
Sure!
Fixed it in https://github.com/vircadia/Blender-Metaverse-Addon/pull/1/commits/b858ad1c0a74555dd85a457c84e5b91d830e9122